docs/content/concepts/cryptography/nautilus/nautilus-seal.mdx

```diff
@@ -0,0 +1,31 @@
+---
+title: Seal Integration with Nautilus
+description: Learn about using Seal for secure secret use within Nautilus enclaves.
+keywords: [ seal, seal secret management, seal secrets, nautilus, enclaves ]
+---
+
+The Nautilus and [Seal](https://seal-docs.wal.app) integration pattern allows Nautilus enclaves to securely fetch public keys and identity-based decryption keys from Seal key servers. These keys can be used to encrypt or decrypt sensitive data directly inside the enclaves, ensuring that data is protected end-to-end, both at rest and in use.
```

Comment:
`securely fetch public keys` i dont think this makes sense. `securely fetch identity-based encryption` sounds good.